src/App/App.js

```diff
@@ -1,25 +1,21 @@
-import logo from '../logo.svg';
+import React from 'react';
+import { Route } from 'react-router-dom';
 import './App.css';
+import Header from '../components/Header/Header';
+import Footer from '../components/Footer/Footer';
+import HomeView from '../views/HomeView/HomeView';
+import ProfileView from '../views/ProfileView/ProfileView';
+
+const App = () => {
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <>
+      <Header />
+      <Route exact path='/' component={HomeView} />
+      <Route exact path='/profile' component={ProfileView} />
+      <Footer />
+    </>
```

Comment:
I'm glad you suggested this approach, @colefiscus - I really like the clean simplicity of this for our App component and the idea of letting either view of this fairly small app contain what needs to be rendered respectively between the Header and Footer